package/DataTableHeaderCell.tsx

```diff
@@ -101,32 +121,31 @@ export default function DataTableHeaderCell<T>({
       onClick={sortAction}
       onKeyDown={(e) => e.key === 'Enter' && sortAction?.()}
     >
-      {sortable || sortStatus?.columnAccessor === accessor ? (
-        <Group className={classes.sortableColumnHeaderGroup} position="apart" noWrap>
-          <Box className={cx(classes.columnHeaderText, classes.sortableColumnHeaderText)} title={tooltip}>
-            {text}
-          </Box>
-          {sortStatus?.columnAccessor === accessor ? (
-            <Center
-              className={cx(classes.sortableColumnHeaderIcon, {
-                [classes.sortableColumnHeaderIconRotated]: sortStatus.direction === 'desc',
-              })}
-              role="img"
-              aria-label={`Sorted ${sortStatus.direction === 'desc' ? 'descending' : 'ascending'}`}
-            >
-              {sortIcons?.sorted || <IconArrowUp size={14} />}
-            </Center>
-          ) : (
-            <Center className={classes.sortableColumnHeaderUnsortedIcon} role="img" aria-label="Not sorted">
-              {sortIcons?.unsorted || <IconArrowsVertical size={14} />}
-            </Center>
-          )}
-        </Group>
-      ) : (
-        <Box className={classes.columnHeaderText} title={tooltip}>
+      <Group className={classes.sortableColumnHeaderGroup} position="apart" noWrap>
```

Comment:
Made rendering of the `<Group>` unconditional now that is required for multiple different components that can show up independently from one another. It is however a breaking change…